git-integration-for-jira-self-managed/App-Support-gij-self-managed.md

```diff
@@ -21,6 +21,18 @@ The Git Integration for Jira app currently supports **Jira** **8.0** to **9.11**
     </div>
 </div>
 
+<div class="bbb-callout bbb--alert">
```

Comment:
IMHO: the block is redundant. It lists just one bug existing in Jira 8.0 There are a lot of other bugs. it clutters up the text.
IMHO: I'd extended the next block "Atlassian Support End of Life" and add the next in bold

```
Starting v2.13.0+, we are dropping support for Java 7, Jira 6.3 and 6.4.

**Starting v4.12+, we are dropping support for Java 8.0-8.11.**
```

Reply:
I mean, you shouldn't use bold. I just used bold to highlight the new phrase.

Reply:
Thank you, Irina,  for your input and suggestions.